src/archivematicaCommon/lib/lazy_paged_sequence.py

```diff
@@ -0,0 +1,48 @@
+class LazyPagedSequence(object):
```

Comment:
Let's use the dependency; this is only vendored because I hadn't published it yet the last time this was touched.